en/developer_guide/multiPanel.rst

```diff
@@ -0,0 +1,81 @@
+Add panel to an administration view
+===================================
+
+Context
+-------
+
+When working with administration back office, sometimes you need to add your own forms or contents to the existing one.
+Open Orchestra implements a system to easily add form to your administration page in a new panel.
+
+Panel System
+~~~~~~~~~~~~
+
+To add a new panel, you have to had a ``_self_panel`` link to the facade of the page.
+
+.. code-block:: php
+
+    $facade->addLink("_self_panel_" + $position + "_" + $panelTitle,
+        $route);
+
+The link name is a key to choose position and name of your panel:
```

Comment:
What is the link name?

You should either : 
 - introduce this concept
 - say what it refers to